### DISCUSSION TITLE
How to delete song from playlist?

### DISCUSSION THREAD

**Discussion by jtmr05** [hrkfdn/ncspot#1122] [Q&A]
Title says it all. In library view, I navigate to the track I want to delete and hit `d`... but nothing seems to happen. Help?

**Comment by Bettehem** [comment]
Hi, it seems like you are already using the correct key. `d` should delete the selected track.
I opened an issue regarding this here: https://github.com/hrkfdn/ncspot/issues/1152

**Comment by jtmr05** [comment]
Thank you. It seems I'm having a similar experience as you. In certain situations I'm able to indeed remove the selected track from a playlist but more often than not it doesn't seem to work.
